legend-engine-pure-runtime-execution/pom.xml

```diff
@@ -147,6 +147,11 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-pure-platform-dsl-mapping-java</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I believe runtime-execution module is expected to be light weight. Should this dependency be added to ide-light module? or where this test executes in pipeline?

Reply:
Moved this dependency to ide light